hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/connector/HazelcastWriters.java

```diff
@@ -328,7 +293,7 @@ public String toString() {
         }
     }
 
-    public static class EntryProcessorWriter<T, K, V> extends AbstractProcessor {
+    static final class EntryProcessorWriter<T, K, V> extends AbstractProcessor {
```

Comment:
This class can be private if you move the `EntryProcessorWriterSupplier` class nested in it out. Or when you remove the import and reference the `EntryProcessorWriterSupplier` as `EntryProcessorWriter.EntryProcessorWriterSupplier`.